pkg/k8shandler/visualization.go

```diff
@@ -432,31 +436,55 @@ func (clusterRequest *ClusterLoggingRequest) createOrUpdateKibanaConsoleExternal
 		return err
 	}
 
-	consoleExternalLogLink := NewConsoleExternalLogLink(
-		"kibana",
+	consoleExternalLogLinkForAdmin := NewConsoleExternalLogLink(
+		"kibana-admin",
 		cluster.Namespace,
 		"Show in Kibana",
 		strings.Join([]string{kibanaURL,
-			"/app/kibana#/discover?_g=(time:(from:now-1w,mode:relative,to:now))&_a=(columns:!(kubernetes.container_name,message),query:(query_string:(analyze_wildcard:!t,query:'",
+			"/app/kibana#/discover?_g=(time:(from:now-1w,mode:relative,to:now))&_a=(columns:!(kubernetes.container_name,message),index:",
+			"'.operations.*'",
+			",query:(query_string:(analyze_wildcard:!t,query:'",
```

Comment:
I'm still unsure that we want to have this query for the `.operations.*` index pattern.
I guess i'd like to know what we seek to accomplish by having an external link for admins -- is it to see specific container logs? see all container logs? see all infra logs that include containers and node logs?

Reply:
My understanding is that when this external log link is clicked then all the context is available. This means all project, pod and container are selected. If we miss any of these then the list of logs seen in the console UI can be different from what will be filtered in Kibana (for example if there are more containers, like app and proxy side car).

Reply:
I understand that, I'm wondering what our intent is with the admin link specifically. I can see it providing one of a few things and wanted to have that clarified:
* see specific container logs
* see all container logs
* see all infra logs that include containers and node logs

Reply:
> I believe the console will present one link or the other based upon a namespace regex

That is correct, I tested this and I can confirm that.

But the important part is in which context (i.e. **WHEN**) the link is displayed. I am aware of only a single case and that is when you navigate to `Pods > Pod Details` section and select the `Logs` tab. I am not aware of any other situation - @spadgett Is there any other?

In this context user have selected all of `Namespace`, `Pod` and `Container`.
And more specifically, this context already presents user with log stream window. My understanding is that the link "Show in Kibana" should present user with exactly the same set of logs:

- **For non-admin** user this is clear. We need to pass to Kibana `namespace`, `pod` and `container` for filtering and use index pattern for given project (this is what the PR does now).

- **For admin** we can do the same except the index pattern will be the `project.*` (hardcoded).

Does it make sense?
WDYT?

Reply:
In other words, when logged in as **the cluster admin** we see the following log stream in console:
![Screenshot 2020-02-05 at 15 41 32](https://user-images.githubusercontent.com/205174/73851653-23878180-482e-11ea-9195-f1631885e355.png)

And upon clicking the "Show in Kibana" link I would expect that user (still cluster admin) should see the same logs:
![Screenshot 2020-02-05 at 15 41 47](https://user-images.githubusercontent.com/205174/73851775-5cbff180-482e-11ea-82cc-a46aefc04f3c.png)
(logs need to be reverse ordered by time ^^)

Reply:
> 
> * **For non-admin** user this is clear. We need to pass to Kibana `namespace`, `pod` and `container` for filtering and use index pattern for given project (this is what the PR does now).

Correct

> * **For admin** we can do the same except the index pattern will be the `project.*` (hardcoded).
> 

This depends.  If I am coming from an 'infra' namespace then it would be '.operations'  If I am coming from a non-infra namespace it would be 'project.*'

Reply:
@jcantrill 
> If I am coming from an 'infra' namespace then it would be '.operations'

What would be the correct approach to learn if user comes from "infra" namespace?
How about the following:

```
$ oc get projects | grep infra
openshift-infra                                                        Active
openshift-kni-infra                                                    Active
openshift-openstack-infra                                              Active
openshift-ovirt-infra                                                  Active
```

Reply:
> @jcantrill
> 
> > If I am coming from an 'infra' namespace then it would be '.operations'
> 
> What would be the correct approach to learn if user comes from "infra" namespace?
> How about the following:
> 
> ```
> $ oc get projects | grep infra
> openshift-infra                                                        Active
> openshift-kni-infra                                                    Active
> openshift-openstack-infra                                              Active
> openshift-ovirt-infra                                                  Active
> ```

Infra means: any namespace that is openshift*, kube*, default